web_widget_numeric_step/static/src/xml/numeric_step.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+Copyright 2019 GRAP - Quentin DUPONT
+Copyright 2020 Tecnativa - Alexandre Díaz
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<template>
+    <t t-name="web_widget_numeric_step" owl="1">
+        <div
+            class="d-flex widget_numeric_step"
+            t-on-mouseleave="_onFocusOutInput"
+            t-on-mouseenter="_onFocusInput"
+        >
+            <div class="input-group-prepend d-none widget_numeric_step_btn">
+                <button
+                    class="fa fa-minus btn btn-default btn_numeric_step"
+                    aria-label="Minus"
+                    title="Minus"
+                    type="button"
+                    data-mode="minus"
+                    t-on-click="_onStepClick"
+                />
+            </div>
+            <input
+                t-att-id="props.id"
+                t-ref="numpadDecimal"
+                t-att-placeholder="props.placeholder"
+                t-att-type="props.inputType"
+                inputmode="numeric"
```

Comment:
The problem with 'numeric' mode is that in mobile you lost some features like use 'formulas' (=2+7*7).